docs/server.js

```diff
@@ -1,37 +0,0 @@
-const { parse } = require('url');
-const next = require('next');
-const { createServer } = require('http');
-const app = next({ dev: true });
-const port = 3000;
-const handle = app.getRequestHandler();
-
-app.prepare().then(() => {
-  createServer((req, res) => {
-    // Enable CORS in development
```

Comment:
Figure out why this was added by looking at the blame history and go through the test plan listed in that commit.